libfoundation/include/foundation.h

```diff
@@ -2051,8 +2051,9 @@ MC_DLLEXPORT real64_t MCNumberFetchAsReal(MCNumberRef number);
 
 MC_DLLEXPORT bool MCNumberParseOffsetPartial(MCStringRef p_string, uindex_t offset, uindex_t &r_chars_used, MCNumberRef &r_number);
 
-MC_DLLEXPORT bool MCNumberParseOffset(MCStringRef p_string, uindex_t offset, uindex_t char_count, MCNumberRef &r_number);
+MC_DLLEXPORT bool MCNumberParseOffset(MCStringRef p_string, uindex_t offset, uindex_t char_count, bool p_integer_only, MCNumberRef &r_number);
```

Comment:
Is there a reason you've added the flag to the public function? We have to be careful changing anything which is MC_DLLEXPORT as it could be bound to by an LCB module.

Reply:
Currently `MCNumberParseOffset` is called in `MCNumberParse` and `MCNumberParseInteger`, so I added the flag to differentiate between these 2. But I could do so by calling the private `__MCNumberParseOffset` instead, so I'll change this.